Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```
@@ -38,14 +38,11 @@
         variation: 0.125
     activatedDamage:
         types:
-            Slash: 8
-            Heat: 10
-            Structural: 20
+            Slash: 12
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 10 but found 12

(indentation)